util/run_manual_tests.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
nit: Can you add a stringdoc describing how this script is used?

Reply:
Done